Sources/Appcast/SUAppcastItem.swift

```diff
@@ -602,6 +602,15 @@ public class SUAppcastItem {
         let rolloutIntervalString = dict[SUAppcastElement.PhasedRolloutInterval] as? String
         self.phasedRolloutInterval = Int(rolloutIntervalString ?? "")
         
+        // Sparkle 1.0.0 supports `<enclosure shortVersionString="">` attribute
+        var shortVersionString = enclosure?[SUAppcastAttribute.ShortVersionString] as? String
+        if shortVersionString == nil {
+            // Sparkle 2.0.0 supports `<sparkle:shortVersionString>` element
```

Comment:
Other comments also contain definite and indefinite articles, so we should stick to that grammatical style:

```suggestion
        // Sparkle 1.0.0 supports the `<enclosure shortVersionString="">` attribute
        var shortVersionString = enclosure?[SUAppcastAttribute.ShortVersionString] as? String
        if shortVersionString == nil {
            // Sparkle 2.0.0 supports the `<sparkle:shortVersionString>` element
```